### PR TITLE
Move personal links into about me

### PR DESCRIPTION
**Pull request by mollygraton**
Also changing background color of the site.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html
Comment (cisxingg) [inline portfolio/src/main/webapp/index.html]: Though this looks super clean and nice, the style guide actually wants us to use a 2-space indentation:
https://google.github.io/styleguide/htmlcssguide.html#Indentation

File: portfolio/src/main/webapp/style.css
Comment (cisxingg) [inline portfolio/src/main/webapp/style.css]: Same nit as above :)

**Review by cisxingg — CHANGES_REQUESTED** [review]
LG in general, just need to fix a couple styling issues.

**Review by cisxingg — APPROVED** [review]
LG, get reviews from Warfa please before merging :)

**Review by warfa-google — APPROVED** [review]
LGTM

**Review by warfa-google — APPROVED** [review]

**Pull request merged**
